app/controllers/admin/applications_controller.rb

```diff
@@ -0,0 +1,17 @@
+class Admin::ApplicationsController < ApplicationController
+  def show
+    @application = Application.find(params[:id])
+  end
+
+  def update
```

Comment:
See note in routes about this action -- this update could live in your PetApplicationsController